int64 task_id
int64 robot_id
int64 bid_value